lib/internal/source_map.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const _sourceMapCache = new Map();
```

Comment:
Why is this underscored? I don't think we usually do this for module-scoped variables?